specification/cosmos-db/resource-manager/readme.md

```diff
@@ -30,6 +30,21 @@ openapi-type: arm
 tag: package-2021-01
 ```
 
+These settings apply only when `--tag=package-2021-03-preview` is specified on the command line.
```

Comment:
Can you make this default tag everywhere. I suspect that would pass the CI.

Reply:
Changed the `tag: package-2021-01` on line 30 to `tag: package-2021-03-preview`.